src/ru/animelib/src/eu/kanade/tachiyomi/animeextension/ru/animelib/Animelib.kt

```diff
@@ -0,0 +1,408 @@
+package eu.kanade.tachiyomi.animeextension.ru.animelib
+
+import android.app.Application
+import android.widget.Toast
+import androidx.preference.EditTextPreference
+import androidx.preference.ListPreference
+import androidx.preference.MultiSelectListPreference
+import androidx.preference.PreferenceScreen
+import androidx.preference.SwitchPreferenceCompat
+import eu.kanade.tachiyomi.animesource.ConfigurableAnimeSource
+import eu.kanade.tachiyomi.animesource.model.AnimeFilterList
+import eu.kanade.tachiyomi.animesource.model.AnimesPage
+import eu.kanade.tachiyomi.animesource.model.SAnime
+import eu.kanade.tachiyomi.animesource.model.SEpisode
+import eu.kanade.tachiyomi.animesource.model.Track
+import eu.kanade.tachiyomi.animesource.model.Video
+import eu.kanade.tachiyomi.animesource.online.AnimeHttpSource
+import eu.kanade.tachiyomi.network.GET
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Animelib : ConfigurableAnimeSource, AnimeHttpSource() {
+
+    override val name = "Animelib"
+
+    override val lang = "ru"
+
+    override val supportsLatest = true
+
+    private val domain = "anilib.me"
+    override val baseUrl = "https://$domain/ru"
+    private val apiUrl = "https://api.lib.social/api"
+
+    private val json = Json {
+        ignoreUnknownKeys = true
+    }
```

Comment:
```suggestion
    private val json: Json by injectLazy()
```

Reply:
Removed entirely, because all json parsing is done by `Response.parseAs<T>`